hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconControllerModule.java

```diff
@@ -96,6 +106,34 @@ protected void configure() {
         .to(ReconStorageContainerManagerFacade.class).in(Singleton.class);
   }
 
+  @Provides
+  @Singleton
```

Comment:
AFAIK, DAOs are in-expensive to create so enforcinf Singleton's might be an anti-pattern.

Reply:
Fixed.